### PR TITLE
Android - iOS Feature Comparison

### PR DESCRIPTION
**Pull request by dshokouhi**
As requested on Discord we thought it would be a nice thing to add a feature comparison table.  I hope I captured everything properly :) took some time to dig through the pages.  Tried to also simplify some of the titles.

The notification part seems a little redundant but I wanted to make sure all supported features are here and make sense. If anyone has any suggestions to make it look better I'm all for it :)

I also purposely left out notification grouping as thats more of a core feature rather than an app feature.

### CODE REVIEW

File: docs/core/index.md [hunk 1]
```diff
@@ -6,3 +6,55 @@ id: 'core'
...
+| [Color](../notifications/basic.md#notification-color) | ✔ |  |
+| [Critical](../notifications/critical.md) | ✔ | ✔ |
+| [Dynamic Attachments](../notifications/dynamic-content.md) |  | ✔ |
+| [Image](../notifications/attachments.md) | ✔ | ✔ |
+| [Message](../notifications/basic.md) | ✔ | ✔ |
+| [Presentation](../notifications/basic.md#controlling-how-a-notification-is-displayed-when-in-the-foreground) |  | ✔ |
+| [Sound](../notifications/sounds.md) |  | ✔ |
+| [Sticky](../notifications/basic.md#sticky-notification) | ✔ |  |
+| [Subtitle](../notifications/basic.md#subtitle) |  | ✔ |
+| [Thread-id](../notifications/basic.md#thread-id-grouping-notifications) |  | ✔ |
```
Comment (robbiet480) [inline docs/core/index.md]:
```suggestion
| [Threads](../notifications/basic.md#thread-id-grouping-notifications) |  | ✔ |

```

File: docs/core/index.md [hunk 2]
```diff
@@ -6,3 +6,55 @@ id: 'core'
...
+| [URL Handler](../integrations/url-handler.md) |  | ✔ |
+| [X-Callback-URL](../integrations/x-callback-url.md) |  | ✔ |
+
+| Notifications | ![android](/assets/android.svg) | ![iOS](/assets/apple.svg) |
+| ------ | ------ | ------ |
+| [Actionable](../notifications/actionable.md) | ✔ | ✔ |
+| [Badge](../notifications/basic.md#badge) |  | ✔ |
+| [Click Action](../notifications/basic.md#notification-click-action) | ✔ |  |
+| [Color](../notifications/basic.md#notification-color) | ✔ |  |
+| [Critical](../notifications/critical.md) | ✔ | ✔ |
```
Comment (robbiet480) [inline docs/core/index.md]:
```suggestion
| [Critical Alerts](../notifications/critical.md) | ✔ | ✔ |

```

File: docs/core/index.md [hunk 3]
```diff
@@ -6,3 +6,55 @@ id: 'core'
...
+| [Sound](../notifications/sounds.md) |  | ✔ |
+| [Sticky](../notifications/basic.md#sticky-notification) | ✔ |  |
+| [Subtitle](../notifications/basic.md#subtitle) |  | ✔ |
+| [Thread-id](../notifications/basic.md#thread-id-grouping-notifications) |  | ✔ |
+| [Title](../notifications/basic.md) | ✔ | ✔ |
+| [Video](../notifications/attachments.md) |  | ✔ |
+| [Replaceable Notifications](../notifications/basic.md#replacing-notifications) | ✔ | ✔ |
+| [Request Location Updates](../notifications/location.md) | ✔ | ✔ |
+
+
```
Comment (robbiet480) [inline docs/core/index.md]:
```suggestion

```

File: docs/core/index.md [hunk 4]
```diff
@@ -6,3 +6,55 @@ id: 'core'
...
+| [Message](../notifications/basic.md) | ✔ | ✔ |
+| [Presentation](../notifications/basic.md#controlling-how-a-notification-is-displayed-when-in-the-foreground) |  | ✔ |
+| [Sound](../notifications/sounds.md) |  | ✔ |
+| [Sticky](../notifications/basic.md#sticky-notification) | ✔ |  |
+| [Subtitle](../notifications/basic.md#subtitle) |  | ✔ |
+| [Thread-id](../notifications/basic.md#thread-id-grouping-notifications) |  | ✔ |
+| [Title](../notifications/basic.md) | ✔ | ✔ |
+| [Video](../notifications/attachments.md) |  | ✔ |
+| [Replaceable Notifications](../notifications/basic.md#replacing-notifications) | ✔ | ✔ |
+| [Request Location Updates](../notifications/location.md) | ✔ | ✔ |
```
Comment (robbiet480) [inline docs/core/index.md]: 🔠

Comment (dshokouhi) [reply docs/core/index.md]: haha oops

**Comment by dshokouhi** [comment]
Didn't consider another one for location updates....that should help clarify some differences I think

**Pull request merged**